tests/components/mpd/test_config_flow.py

```diff
@@ -42,6 +42,28 @@ async def test_full_flow(
     }
     assert len(mock_setup_entry.mock_calls) == 1
 
+    # Setup a second entry with same host, but different port.
```

Comment:
Rather extend `test_existing_entry`

Reply:
Same code? Or would you prefer a different approach?

Reply:
Updated with the same code, but happy to approach it a different way if wanted

Reply:
Oh wait, now that i think of it, i think only the current tests has to be checked if it still works, we dont really have to extend the test, sorry for that

Reply:
Do we want to remove it? It does test a new behavior that might break in the future without a test

Reply:
Yea we can add a test for that (but I want to mention that it's not a blocker for me if it's missing), but then I'd rather see a test similar like test_existing_entry and in a separate test :)

Reply:
It was getting messy to get the test to work, so removed it and squashed the commits